QPSK/Modulator.py

```diff
@@ -0,0 +1,27 @@
+import numpy
+
+
+def createSignalI(bit, carrierFreq, fi, time):
+    return bit * numpy.cos(2 * numpy.pi * carrierFreq * time + fi)
```

Comment:
zastanów się nad dodaniem jakiegoś psf, bo finalnie chyba będziemy potrzebować tutaj

Reply:
dodany task to Trello